backend/codesirius/problems/serializers/reference_solution.py

```
@@ -38,26 +39,31 @@
-            )
-            return ReferenceSolution.objects.create(
-                **validated_data, problem=self.context["problem"]
-            )
-        except Exception as e:
-            logger.error(f"Failed to create a new reference solution: {e}")
-            raise serializers.ValidationError(
-                "Failed to create a new reference solution"
-            )
+    class Meta:
+        model = ReferenceSolution
+        fields = ["id", "problemId", "code", "languageId", "verdict"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)